src/AnimatedFade.tsx

```diff
@@ -1,52 +1,108 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useCallback, useEffect, useRef } from 'react';
 import { Animated, ViewStyle } from 'react-native';
+import useCurrent from './hooks/useCurrent';
 
 type Props = {
-  fadeOnMount?: boolean;
+  /**
+   * auto-trigger fade-in?
+   */
   triggerFade?: boolean;
-  startingOpacity?: number; // 0 - 1.00
-  endingOpacity?: number; // 0 - 1.00
-  duration?: number; // ms
-  delay?: number; // ms
+
+  /**
+   * starting opacity value (0 - 1.00)
+   */
+  opacityStart?: number;
+
+  /**
+   * ending opacity value (0 - 1.00)
+   */
+  opacityEnd?: number;
+
+  /**
+   * animation duration (in ms)
+   */
+  duration?: number;
+
+  /**
+   * delay before animation start (in ms)
+   */
+  delay?: number;
   style?: ViewStyle;
+
+  /**
+   * callback function called once fadeIn animation completes
+   */
   onEnd?: () => void;
+
+  /**
+   * should this animation repeat?
+   */
+  loop?: boolean;
 };
 
 const AnimatedFade: React.FC<Props> = ({
-  fadeOnMount = true,
```

Comment:
`fadeOnMount` and `triggerFade` did exactly the same thing, so I removed this redundancy. There is no case where a single prop could not achieve the same functionality.

e.g.

```
const [shouldFadeIn, setShouldFadeIn] = setState(false);
<AnimatedFade fadeOnMount={false} triggerFade={shouldFadeIn}> ... </AnimatedFade>
<Button onPress={() => { setShouldFadeIn(true); }>Fade Me!</Button>
```

has zero difference to the following when `fadeOnMount` prop is removed:

```
const [shouldFadeIn, setShouldFadeIn] = setState(false);
<AnimatedFade triggerFade={shouldFadeIn}> ... </AnimatedFade>
<Button onPress={() => { setShouldFadeIn(true); }>Fade Me!</Button>
```

Reply:
Love it @townofdon! Works great in my testing